golem-worker-executor-base/src/durable_host/mod.rs

```diff
@@ -1217,15 +1209,22 @@ impl<Ctx: WorkerCtx> PrivateDurableWorkerState<Ctx> {
 
     async fn get_oplog_entry_exported_function_invoked(
         &mut self,
-    ) -> Result<Option<(String, Vec<Value>, InvocationKey, Option<CallingConvention>)>, GolemError>
-    {
+    ) -> Result<
+        Option<(
+            String,
+            Vec<Value>,
+            IdempotencyKey,
+            Option<CallingConvention>,
```

Comment:
Minor: Could have been better to be a struct?

Reply:
This is not changed in this PR, you just see here because of the rename